idaes/core/surrogate/keras_surrogate.py

```diff
@@ -263,7 +263,7 @@ def save_to_folder(self, keras_folder_name):
               The name of the folder to contain the Keras model and additional
               IDAES metadata
         """
-        self._keras_model.save(keras_folder_name)
+        self._keras_model.save(os.path.join(keras_folder_name, "idaes_keras_model.keras"))
```

Comment:
Should the file name be hard-coded here, or should we make this an input from the user (either optional or required).

Reply:
I support letting users pass a string for the file name, so something like

```suggestion
        self._keras_model.save(os.path.join(keras_folder_name, keras_file_name + ".keras"))
```

where `keras_file_name` is an input to the save method. I don't know if we still need to specify the folder name, or if a single string for the entire path is sufficient.

Reply:
The folder name is used to save/load some model weights and some idaes specific information separate from the filename.  But I think the file name should be rename-able as you say -- making that change now